enclave-runtime/Cargo.toml

```diff
@@ -110,21 +110,19 @@ itp-utils = { path = "../core-primitives/utils", default-features = false, featu
 its-sidechain = { path = "../sidechain/sidechain-crate", default-features = false, features = ["sgx"] }
 
 # substrate deps
-frame-support = { default-features = false, git = "https://github.com/paritytech/substrate.git", branch = "polkadot-v0.9.20" }
-sp-core = { default-features = false, features = ["full_crypto"], git = "https://github.com/paritytech/substrate.git", branch = "polkadot-v0.9.20" }
-sp-finality-grandpa = { default-features = false, git = "https://github.com/paritytech/substrate.git", branch = "polkadot-v0.9.20" }
-sp-runtime = { default-features = false, git = "https://github.com/paritytech/substrate.git", branch = "polkadot-v0.9.20" }
-sp-std = { default-features = false, git = "https://github.com/paritytech/substrate.git", branch = "polkadot-v0.9.20" }
-sp-version = { default-features = false, git = "https://github.com/paritytech/substrate.git", branch = "polkadot-v0.9.20" }
-sp-application-crypto = { default-features = false, git = "https://github.com/paritytech/substrate.git", branch = "polkadot-v0.9.20" }
+frame-support = { default-features = false, git = "https://github.com/paritytech/substrate.git", branch = "polkadot-v0.9.24" }
+sp-core = { default-features = false, features = ["full_crypto"], git = "https://github.com/paritytech/substrate.git", branch = "polkadot-v0.9.24" }
+sp-finality-grandpa = { default-features = false, git = "https://github.com/paritytech/substrate.git", branch = "polkadot-v0.9.24" }
+sp-runtime = { default-features = false, git = "https://github.com/paritytech/substrate.git", branch = "polkadot-v0.9.24" }
+sp-std = { default-features = false, git = "https://github.com/paritytech/substrate.git", branch = "polkadot-v0.9.24" }
+sp-version = { default-features = false, git = "https://github.com/paritytech/substrate.git", branch = "polkadot-v0.9.24" }
+sp-application-crypto = { default-features = false, git = "https://github.com/paritytech/substrate.git", branch = "polkadot-v0.9.24" }
 
 [patch.crates-io]
-log = { git = "https://github.com/mesalock-linux/log-sgx" }
```

Comment:
@mullefel I'd like to ask for your "Good to go" for that one. 
Substrate has updated to `log v0.4.17`, whereas mesalock is still on `v0.4.14`. Since patches only work for newer or equal version numbers, this patch didn't work any more for substrate libraries and resulted in a compile error.

We have two solutions:
a) update mesalock to `v0.4.17` ourselves (probably effectively fork it, as mesalock is not really maintained any more)
b) remove mesalock log, as log itself is no-std compatible. @gaudenzkessler has tested it, and it's still working.

What's do you think?

Reply:
This is just what I've noticed last Thursday with the Ajuna worker: When compiled with `SGX_MODE=HW`, the regular `log` crate will not produce any log output. So we absolutely need the SGX port - we probably even need to introduce it in all our crates in the worker (see #819 ). So if a) and b) are our only options, then it has to be a) IMO.

Reply:
After reviewing and testing, this seems to be an Ajuna specific problem. On our current worker, the std `log` crates works fine, also for `HW` mode.